lib/transport/data_validator/server.ex

```diff
@@ -24,6 +24,10 @@ defmodule Transport.DataValidator.Server do
     GenServer.call(:publisher, {:apply_async, "tasks.perform", [args: [url]]})
   end
 
+  def get_task(task_id) do
+    GenServer.call(:publisher, {:get_task_meta, task_id})
```

Comment:
Why `:get_task_meta`? That's celery's internal naming, for us is just a task.

Reply:
here we are doing celery internal things. I'm trying to stick to the original implementation to make it more easily debuggable.